code/game/objects/items/breakable.dm

```diff
@@ -0,0 +1,129 @@
+//Breakable items
+
+//TODO:
+//multiples of a single fragment
+//add damaged text into the general examine proc
+//check slimes
+//add thrown response
+//projectiles?
+//kicking and biting?
+//change "hit" to something else?
+//hit sounds
+//punching hurting hands and stuff?
+//use kicking code
+//generalize out of devices
+
+/obj/item/device
+
+	//Destructability parameters:
+	var/breakable = 0 //1: breakable (by smashing it using the following flags
```

Comment:
Use TRUE and FALSE whenever the expected values are only 1 and 0.
Also, isn't this redundant because `breakable_flags = 0` exists?

Reply:
I was thinking a situation could arise where someone would want to toggle an item's breakability off without setting the flags to 0.

Reply:
I think I'll remove `breakable` and instead add a list of excluded items that bypass the breaking so that specific harm intent interactions can still be performed on a breakable item while the item still remains generally breakable.